### ISSUE TITLE
latest version acts like --no is set, won't prompt for yes or create any files

### ISSUE THREAD

**Issue by VictorBargains** [paul-gauthier/aider#585]
### Issue

I am trying a new install of aider on my work system, starting with the latest version. I have used a previous version on my personal computer so I'm not yet 100% sure if this is a new version problem or a new system problem. `Edit: tried with 0.30.0 and did not find this problem`

Environments between the two (VSCode, Git Bash, Windows 11) are identical, so I am leaning towards this being a new version problem.

What happens is that it never asks me yes/no to make files. It just says no and goes back to the prompt. The aider chat log shows a double instance of the "no" prompt line.

I tried passing `--yes` as an arg and it did not change the behavior. It's as if --no was hard coded instead.

![Screenshot 2024-05-02 115836](https://github.com/paul-gauthier/aider/assets/2579047/c6ebb808-a416-473a-a2e4-bee38680d027)

### Version and model info

# aider chat started at 2024-05-02 10:47:33

> C:\Users\vandersen\.virtualenvs\parser-mGiWKD-F\Scripts\aider  
> Aider v0.31.0  
> Models: gpt-4-1106-preview with udiff edit format, weak model gpt-3.5-turbo  
> Git repo: ..\..\..\..\..\.git with 8 files  
> Repo-map: using 1024 tokens  
> VSCode terminal detected, pretty output has been disabled.  
> Use /help to see in-chat commands, run with --help to see cmd line args  
> Note: in-chat filenames are always relative to the git working dir, not the current working dir.  
> Cur working dir: C:\Users\vandersen\source\repos\SMWParser\py_simpl_mermaid\src\simpl\windows\gui\parser  
> Git working dir: C:\Users\vandersen\source\repos\SMWParser\py_simpl_mermaid

**Comment by paul-gauthier** [comment]
Thank you for reporting this! I just published a new release v0.31.1 which fixes this issue. You should be able to upgrade via the command below. It sometimes takes pypi a few minutes to pick up the newest release.

```
pip install --upgrade aider-chat
```

Or you can install the latest version from github:

```
python -m pip install --upgrade git+https://github.com/paul-gauthier/aider.git
```

If you have a chance to try it, let me know if it works better for you.

**Comment by VictorBargains** [comment]
New version working. Thanks for the quick turnaround!
